resources/views/reviews.blade.php

```diff
@@ -24,4 +23,20 @@
             </div>
         </graphql>
     </lazy>
+    @else
+    <div class="w-full p-8 mb-4 border rounded bg-white">
+        <div class="flex items-center gap-2 text-secondary">
+            <strong itemprop="name">@lang('No reviews found')</strong>
+            <span class="text-xs">
+                @{{ new Date(Date.now()).toLocaleDateString() }}
```

Comment:
```suggestion
                {{ Carbon::now()->toDateString() }}
```
But why is the date displayed here?

Reply:
As a placeholder, it looks nice